package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
```

Comment:
- [ ] Please ensure that `react-dom` and `babel-loader` packages are in the package.json file

Reply:
Thank you for your feedback,@codepantha